api/src/api/resource.js

```diff
@@ -33,8 +33,17 @@ const addFields = {
 router.get(
   '/',
   errorWrap(async (req, res) => {
-    const { category, subcategory, cost, language, city, sort } = req.query;
-
+    // add page and size
```

Comment:
what is this comment?

Reply:
ohh I think that was to tell myself to add page and size and then I forgot to delete the comment oops